.github/workflows/ci-build.yml

```
@@ -172,4 +172,56 @@ jobs:
         if: always()
         with:
           name: ledfx-log-${{ runner.os }}-${{ runner.arch }}-py${{ matrix.python-version }}.log
-          path: ${{ github.workspace }}/debug_config/ledfx.log
\ No newline at end of file
+          path: ${{ github.workspace }}/debug_config/ledfx.log
+  
+  docker-build:
+    runs-on: ubuntu-latest
+    name: Build LedFx Docker Image
+    steps:
+    - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)